tests/testthat/test-test.R

```diff
@@ -141,3 +145,12 @@ test_that("runTests runs as expected without rewiring", {
   ))
   expect_s3_class(df, "shiny_runtests")
 })
+
+
+test_that("app template works with runTests", {
+  tempTemplateDir <- file.path(tempdir(), paste0("shinyAppTemplate-", floor(runif(1) * 10000)))
+  shinyAppTemplate(tempTemplateDir, "all")
```

Comment:
Would be good to test with different settings as well, notably the different combinations of `rdir` and `module`.